class-pinterest-for-woocommerce.php

```diff
@@ -667,6 +668,21 @@ public static function disconnect() {
 
 				// At this point we're disconnected.
 				return true;
+			} catch ( PinterestApiException $e ) {
+				$code = $e->get_pinterest_code();
+
+				if ( PinterestApiException::MERCHANT_NOT_FOUND == $code ) {
```

Comment:
Do we want a strict comparison here, or are we expecting multiple types of values? `get_pinterest_code()` is supposed to return an `int`.

Reply:
We want `int` and only int. Fixing.